src/api/models.py

```diff
@@ -142,11 +142,11 @@ class Persons(models.Model):
     name = models.CharField(max_length=100)
     dob = models.DateField(null=True, blank=True)
     gender = models.IntegerField(default=0, choices=gender_choices)
-    race_ethnicity = ArrayField(models.IntegerField(default=0, choices=race_ethnicity_choices))
+    race_ethnicity = models.IntegerField(default=0, choices=race_ethnicity_choices)
```

Comment:
Why did you change this from an array field? JGCC/DVHRT allow victims/abuser to identify as more than one race